__tests__/app.test.js

```diff
@@ -27,19 +31,62 @@ describe("/api/categories", () => {
         .expect(200)
         .then(({ body }) => {
           expect(Object.keys(body)).toEqual(["categories"]);
-          expect(body.categories.length).toBe(7);
+          expect(body.categories.length).toBe(4);
         });
     });
     test("status 200 - categories have a 'slug' and 'description' property", () => {
       return request(app)
         .get("/api/categories")
         .expect(200)
         .then(({ body }) => {
-          expect(Object.keys(body.categories[0])).toEqual([
-            "slug",
-            "description",
-          ]);
+          const mappedCategories = [];
+          body.categories.map((category) => {
+            mappedCategories.push({
+              slug: category.slug,
+              description: category.description,
+            });
+          });
+          expect(body.categories).toEqual(mappedCategories);
         });
```

Comment:
I am following the logic here as we would get undefined if the keys don't exist. There are other jest matchers available that can help with checking object contents though. Might be worth looking at the docs for this.